### PR TITLE
PWN-6993 - Off-ramp. Design fixes

### PR DESCRIPTION
**Pull request by gslevinkov**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-6963

### CODE REVIEW

**Review by Davran331122 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/moonpay/extensions/SellStubs.kt [hunk 1]
```diff
@@ -38,4 +40,13 @@ object SellStubs {
         moonpayDepositWalletAddress = Constants.SOL_MINT.toBase58Instance(),
         updatedAt = emptyString()
     )
+
+    val TRANSACTION_DETAILS = SellTransactionViewDetails(
```
Comment (Davran331122) [inline app/src/main/java/org/p2p/wallet/moonpay/extensions/SellStubs.kt:44]: Do we use it in code ?, If it was created for developing feature. I think we should remove this from code. Or use just in test package at all )

**Review by Davran331122 — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/moonpay/extensions/SellStubs.kt (hunk 1 above)
Comment (gslevinkov) [reply app/src/main/java/org/p2p/wallet/moonpay/extensions/SellStubs.kt:44]: yes, i use when i need to open some particular sell screens without any proxies like Charles

**Review by NikolaiEmelianov — APPROVED** [review]

File: .github/workflows/android-feature-workflow.yml [hunk 2]
```diff
@@ -46,7 +46,7 @@ jobs:
       - android-lint
 
     runs-on: ubuntu-latest
-    environment: feature
+    environment: production
```
Comment (1sakov) [inline .github/workflows/android-feature-workflow.yml:49]: ? 
Fetch updates from develop please, there are new changes

File: app/src/main/java/org/p2p/wallet/history/ui/token/adapter/holders/HistorySellTransactionViewHolder.kt [hunk 3]
```diff
@@ -59,7 +59,7 @@ class HistorySellTransactionViewHolder(
                 titleStatus = getString(R.string.transaction_history_moonpay_waiting_for_deposit_title)
                 subtitleReceiver = getString(
                     R.string.transaction_history_moonpay_waiting_for_deposit_subtitle,
-                    item.transactionDetails.receiverAddress.cutMiddle()
+                    item.transactionDetails.receiverAddress.cutStart(4)
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/history/ui/token/adapter/holders/HistorySellTransactionViewHolder.kt:62]: Add annotation for magic number or move to constant

File: app/src/main/java/org/p2p/wallet/moonpay/ui/transaction/SellTransactionDetailsPresenter.kt [hunk 4]
```diff
@@ -67,7 +67,7 @@ class SellTransactionDetailsPresenter(
         )
         val receiverBlock = SellTransactionDetailsViewState.ReceiverBlock(
             receiverTitle = resources.getString(R.string.sell_details_send_to),
-            receiverValue = currentTransaction.receiverAddress.cutMiddle(),
+            receiverValue = currentTransaction.receiverAddress.cutMiddle(cutCount = 6),
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/moonpay/ui/transaction/SellTransactionDetailsPresenter.kt:70]: If I remember correctly, cutMiddle cuts 6 by default

**Review by 1sakov — CHANGES_REQUESTED** [review]

File: .github/workflows/android-feature-workflow.yml (hunk 2 above)
Comment (gslevinkov) [reply .github/workflows/android-feature-workflow.yml:49]: that's not me, damn1

**Review by 1sakov — APPROVED** [review]

**Pull request merged**